src/sdk/pynni/nni/counter.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import torch
+import torch.nn as nn
+from torch.nn.modules.conv import _ConvNd
+from torch.nn.modules.batchnorm import _BatchNorm
+from nni.compression.torch.compressor import PrunerModuleWrapper
+import logging
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from thop import profile
+except ImportError:
+    _logger.warning('Please install thop first.')
+    
+def count_flops_params(model: nn.Module, input_size=None, verbose=True):
+    """
+    Count FLOPs and Params of the given model. 
+    This function would identify the mask on the module 
+    and take the pruend shape into consideration. 
+    Note that, we only identify the output shape of the module, 
```

Comment:
this note is not clear, could you explain a little bit more?

Reply:
Sure. I add more explanation, please check it.